tests/conftest.py

```diff
@@ -0,0 +1,65 @@
+import functools
+import logging
+import os
+import os.path
+
+import pytest
+
+from pytest_bdd import given
+from pytest_bdd import parsers
+from pytest_bdd import then
+from pytest_bdd import when
+
+from utils.helper import run_ansible_playbook
+
+# logging.basicConfig(level=logging.DEBUG)
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        '--inventory', help='Inventory where is specified servers for tests'
+    )
+
+
+@pytest.fixture
+def pytestbdd_strict_gherkin():
```

Comment:
Do we really need this?

Reply:
Yes for the installation test to do `Given-When-Then-When-Then`. You're Stuck at `Given-When-Then` with normal scheme

Reply:
OK